config/default/common/config/wv.json/stories/default/sunglint.json

```diff
@@ -5,12 +5,16 @@
       "type": "default",
       "title": "Sunglint",
       "description": "When sunlight reflects off the surface of water at the same angle as the satellite sensor views it you get sunglint!",
```

Comment:
I think this line needs to be reworded or have punctuation added - I had to re-read it a few times as I got stuck near the end.

Reply:
"What's that shining off the water's surface? It's sunglint!"